docs/source/release/v6.13.0/Indirect/New_features/39212.rst

```diff
@@ -0,0 +1 @@
+- Algorithm :ref:`algm-VesuvioPeakPrediction` now supports option `Classical`, used to estimate the lower bounds of the width of the Neutron Compton Profiles when running analysis routines.
```

Comment:
ReStructuredText uses double backticks for monospaced formatting, if that was the intention. 
```suggestion
- Algorithm :ref:`algm-VesuvioPeakPrediction` now supports option ``Classical``, used to estimate the lower bounds of the width of the Neutron Compton Profiles when running analysis routines.
```